dummyserver/server.py

```diff
@@ -232,3 +215,70 @@ def encrypt_key_pem(private_key_pem: trustme.Blob, password: bytes) -> trustme.B
         serialization.BestAvailableEncryption(password),
     )
     return trustme.Blob(encrypted_key)
+
+
+def main():
+    # For debugging dummyserver itself - python -m dummyserver.server
+    from .handlers import TestingApp
+
+    host = "127.0.0.1"
+
+    async def main():
+        io_loop = tornado.ioloop.IOLoop.current()
+        app = tornado.web.Application([(r".*", TestingApp)])
+        server, port = run_tornado_app(app, io_loop, None, "http", host)
```

Comment:
Would you say that assert io_loop == tornado.ioloop.IOLoop.current() is still useful in run_tornado_app?